modules/enabling-additional-enabled-capabilities.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// *post_installation_configuration/cluster-capabilities.adoc
+
+[id="setting_additional_enabled_capabilities_{context}"]
+= Enabling the cluster capabilities by setting additional enabled capabilities
+
+As a cluster administrator, you can enable the cluster capabilities by setting `additionalEnabledCapabilities`.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (oc).
+* You have installed the link:https://stedolan.github.io/jq/[jq package].
+
+.Procedure
+
+. View the enabled capabilities by running the following command:
```

Comment:
How about `View the additional enabled capabilities...` ?

Reply:
makes sense. I will update